src/plugin/cursus/Listener/DataSource/SessionEventsSource.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Claroline Connect package.
+ *
+ * (c) Claroline Consortium <consortium@claroline.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Claroline\CursusBundle\Listener\DataSource;
+
+use Claroline\AppBundle\API\FinderProvider;
+use Claroline\AppBundle\API\Options;
+use Claroline\CoreBundle\Event\DataSource\GetDataEvent;
+use Claroline\CursusBundle\Entity\Event;
+use Claroline\CursusBundle\Entity\Session;
+
+class SessionEventsSource
+{
+    /** @var FinderProvider */
+    private $finder;
+
+    public function __construct(FinderProvider $finder)
+    {
+        $this->finder = $finder;
+    }
+
+    public function getData(GetDataEvent $event)
+    {
+        /*$sessions = $this->finder->search(Session::class, [
+            'filters' => ['workspace' => $event->getWorkspace()->getUuid()],
+        ], [Options::SERIALIZE_MINIMAL]);*/
+
+        $options = $event->getOptions();
+        $options['hiddenFilters']['publicRegistration'] = true;
```

Comment:
This filter does not exist for training events and can be removed. Also remove commented code please.